File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{ int height;
    do
    {
        printf("Height: ");
        scanf("%d",&height);
    }
    while (height<1 || height>8);  
    
    for(int i=1; i<=height; i++)
      {
            for( int j=1; j<=height-i; j++)
            {
                printf(" ");
            }
            for(int k=1; k<=i; k++)
            {
              printf("#");
            }
         
            printf("  ");
     
            for(int l=1; l<=i; l++)
            {
                 printf("#");
             }
        
        printf("\n");

}
}

File: mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        width = height
        if height > 0 and height < 9:
            break

    for i in range(1, height + 1):
        num_hash = i
        num_space = width - num_hash
        print(" " * num_space, end="")
        print("#" * num_hash, end="")
        print()


if __name__ == "__main__":
    main()